start

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+DIR=`dirname "\$0"`
```

Comment:
Why do we use it with the backslash here?

Reply:
I'm sure I copied that from a StackOverflow answer. Maybe it's necessary to escape the `$` since it is inside both double-quotes and backtick-quotes?

Reply:
I tested the currently used approach, and it works fine with and without `\` identically, so I am not sure we need the `\` sign. However the returned `DIR` is always a _relative_ path. If the idea was to use the absolute path, I'd go with [this approach](https://stackoverflow.com/a/4774063/4143531):
```bash
SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```
What do you think, @danielballan?

Reply:
Good catch! I agree.